djangocms_attributes_field/fields.py

```diff
@@ -119,7 +120,7 @@ def get_default(self):
     def get_internal_type(self):
         return 'TextField'
 
-    def contribute_to_class(self, cls, name, **kwargs):
+    def contribute_to_class(self, cls, name, **kwargs):  # pragma: no cover
```

Comment:
Doesn't this get executed during a test run? Or is the problem that it happens during import, rather than actual test runtime? In that case, it might be worth exploring the possibility of testing this separately on an abstract model class created inside a test, or something like that.